garrysmod/lua/menu/problems/problems_pnl.lua

```diff
@@ -48,6 +48,14 @@ function PANEL:Init()
 
 end
 
+function PANEL:Think()
+
+    if ( input.IsKeyDown( KEY_ESCAPE ) ) then
+        self:Remove()
+    end
+
+end
+
```

Comment:
I am not sure how much I like this when you are in-game. Pressing ESC would close both the main menu, AND the problems panel.